parm/config/config.resources.nco.static

```diff
@@ -133,6 +134,7 @@ elif [ $step = "anal" ]; then
     export npe_node_anal=$(echo "$npe_node_max / $nth_anal" | bc)
     export nth_cycle=$npe_node_max
     export npe_node_cycle=$(echo "$npe_node_max / $nth_cycle" | bc)
+    if [[ "$machine" = "WCOSS2" ]]; then export memory_anal="500GB"; fi
```

Comment:
Is this needed? 
The total memory on the WCOSS2 node is 512GB.  If this resource request is not passed, does it explicitly request the full memory or does it do some calculation based on the number of cores requested?

@GeorgeVandenberghe-NOAA might know the answer.